nano/node/common.cpp

```diff
@@ -696,6 +696,21 @@ void nano::keepalive::serialize (nano::stream & stream_a) const
 	}
 }
 
+std::string nano::keepalive::to_string () const
+{
+	std::stringstream stream;
+
+	stream << header.to_string () + "\n";
+
+	for (auto peer = peers.begin (), j (peers.end ()); peer != j; ++peer)
```

Comment:
Please use
```
j = peers.end()
```

we prefer the assignment style of initialisation

Reply:
if I see somewhere else that's not doing the assignment style of initialization, can I change it, or would that be out of the scope of this PR and not approved of